pset-functions.js

```diff
@@ -31,6 +31,73 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+// function that accepts a score for the player and the dealer and determines the winner
+function blackJack(player, dealer) {
+
+// checks to see if the player busts. If so, and the dealer is under 21, the dealer wins. If both bust, return 0.
+if (player > 21){
```

Comment:
Justin, really great use of if/else statements to capture all potential situations! You can probably make your life a little easier by chaining if/else blocks; similar to like they do [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L33-L43).